dist/legacy-tokens/ds4-light.css

```diff
@@ -134,4 +134,39 @@
         var(--color-ai-solid-green-subtle) 0%,
         var(--color-ai-solid-blue-subtle) 154.5%
     );
+    --color-loading-fill: #f2f2f2;
```

Comment:
This may be pedantic, but I'm not sure how I feel about these all being called `color-loading-` instead of something like `color-skeleton-` since there are other components that we use to indicate loading (like progress spinner)

Reply:
I would like to keep it as is, because we need to be consistent with the design token system. IMO, tokens should be independent of specific components but if it makes more sense to add skeleton, it needs to change at DS level.